### PR TITLE
[TACHYON-1117] Add shell command setTTL and unsetTTL

### PR DESCRIPTION
**Pull request by uronce-cc**
Follow up PR for https://github.com/amplab/tachyon/pull/1800

### CODE REVIEW

File: shell/src/main/java/tachyon/shell/TfsShell.java [hunk 1]
```diff
@@ -659,6 +660,29 @@ public void pin(TachyonURI path) throws IOException {
     }
   }
 
+  public void setTTL(TachyonURI path, long ttlMs) throws IOException {
```
Comment (yupeng9) [inline shell/src/main/java/tachyon/shell/TfsShell.java]: Missing Javadoc

File: shell/src/main/java/tachyon/shell/TfsShell.java [hunk 2]
```diff
@@ -659,6 +660,29 @@ public void pin(TachyonURI path) throws IOException {
...
+      TachyonFile fd = mTfs.open(path);
+      SetStateOptions options = new SetStateOptions.Builder().setTTL(ttlMs).build();
+      mTfs.setState(fd, options);
+      System.out.println("TTL of file '" + path + "' was successfully set to " + ttlMs + ".");
+    } catch (TachyonException e) {
+      throw new IOException(e.getMessage());
+    }
+  }
+
+  public void unsetTTL(TachyonURI path) throws IOException {
```
Comment (yupeng9) [inline shell/src/main/java/tachyon/shell/TfsShell.java]: Missing Javadoc

**Comment by yupeng9** [comment]
Thanks for enabling this in the shell command. I added a minor comment.

File: shell/src/main/java/tachyon/shell/TfsShell.java [hunk 3]
```diff
@@ -659,6 +660,29 @@ public void pin(TachyonURI path) throws IOException {
...
+    try {
+      TachyonFile fd = mTfs.open(path);
+      SetStateOptions options = new SetStateOptions.Builder().setTTL(Constants.NO_TTL).build();
+      mTfs.setState(fd, options);
+      System.out.println("TTL of file '" + path + "' was successfully removed.");
+    } catch (TachyonException e) {
+      throw new IOException(e.getMessage());
+    }
+  }
```
Comment (calvinjia) [inline shell/src/main/java/tachyon/shell/TfsShell.java]: Does it make sense to just use 1 method? ie. `setTTL(path, 0)`

Comment (uronce-cc) [inline shell/src/main/java/tachyon/shell/TfsShell.java]: sgtm

File: shell/src/main/java/tachyon/shell/TfsShell.java (hunk 1 above)
Comment (uronce-cc) [inline shell/src/main/java/tachyon/shell/TfsShell.java]: added

**Comment by uronce-cc** [comment]
jenkins, test this please

**Comment by yupeng9** [comment]
LGTM

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**